docs/src/configuration/effects.md

```diff
@@ -89,6 +89,7 @@ Examples of the JSON defining effects
     ],
     "projectile_attributes": [],
     "skills_keys_to_execute": []
+    "skills_to_execute": []
```

Comment:
But actually I would also like to challenge this property. Since it is really similar to the other one I don't see any reason to not expand the other instead of having this one with a slightly different behaviour. This difference will certainly cause problems on the long run

So instead I would rather see a more complex object in `skills_keys_to_execute` to make the differentiator, sort of

```
skills_keys_to_execute: [
  { skill_key: "123", trigger_action: false },
  { skill_key: "987", trigger_action: true}
]
```

So now this `trigger_action` (please try to find a better name) would be the one making the distinction and everything works the same way